test/LUCI_ExportTestReg.ecl

```diff
@@ -7,6 +7,7 @@ IMPORT ML_Core.Types;
 numTrees := 20;
 maxDepth := 255;
 numFeatures := 0; // Zero is automatic choice
+numRecords := 2000; // Max is number of records in datasets/CovTypeDS.ecl (typically 5000)
```

Comment:
If number is typically 5000, why set it to 2000?

Reply:
The comment is not completely clear, but this attribute is used to further restrict the number of records below the max to reduce run time and the size of the export file.   Max is typically 5000, but we don't want to run the test with max records.